src/simplest-rust/do.sh

```diff
@@ -0,0 +1,109 @@
+#!/usr/bin/env bash
```

Comment:
Could use a repo common `do.sh` for these programs and also use a common workspace to consolidate dependencies.

Reply:
ok